docs/source/solving-interface.rst

```diff
@@ -278,10 +278,10 @@ do this, we set the ``pc_type`` to ``'lu'`` and tell PETSc to use a
 "preconditioner only" Krylov method:
 
 .. code-block:: python3
-
+   
    solve(a == L,
          solver_parameters={'ksp_type': 'preonly',
-                            'pc_type': 'lu'})
+                            'pc_type': 'lu', 'pc_factor`: _DEFAULT_DIRECT_SOLVER_PARAMETERS'})
```

Comment:
Please lift the actual values from the source files (described [here](https://github.com/firedrakeproject/firedrake/issues/3763#issuecomment-2695314845)).